xbmc/windowing/gbm/WinSystemGbm.cpp

```diff
@@ -85,6 +85,18 @@ bool CWinSystemGbm::InitWindowSystem()
       ->GetSettings()
       ->GetSetting("videoscreen.limitguisize")
       ->SetVisible(true);
+  CServiceBroker::GetSettingsComponent()
+      ->GetSettings()
+      ->GetSetting(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK)
+      ->SetVisible(false);
+  if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK))
+  {
+    CServiceBroker::GetSettingsComponent()
+        ->GetSettings()
+        ->SetBool(CSettings::SETTING_VIDEOPLAYER_USEDISPLAYASCLOCK, false);
+    CLog::Log(LOGWARNING, "CWinSystemGbm::%s: Setting '%s' is not supported and has been disabled",
```

Comment:
no need for the log really

Reply:
OK, will drop it and replace by an unconditional set(false)